parent/pom.xml

```diff
@@ -5009,6 +5009,9 @@
               <target>${jdk.version}</target>
               <maxmem>512M</maxmem>
               <fork>${compiler.fork}</fork>
+              <compilerArgs>
```

Comment:
I wonder a bit why you added `-parameters` here.

Reply:
Ah okay, would you mind adding a `<!-- -->` comment that this is required by Spring Boot so we wont forget why its there.